spread.yaml

```diff
@@ -66,4 +124,6 @@ suites:
   tests/spread/commands/:
     summary: simple charmcraft commands
   tests/spread/charms/:
-    summary: charm building tests
+    summary: real charm building tests
+  tests/spread/smoketests/:
```

Comment:
there are two options here, use the external tools to install the appropriate packages for this to work under Fedora or restrict the systems here to ubuntu-*